hack/images.yaml

```diff
@@ -16,7 +16,7 @@ kafka-0.23.0-dev.0:
   - "gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0"
 kafka-0.25.1:
   - "ghcr.io/banzaicloud/kafka-operator:v0.25.1"
-  - "ghcr.io/banzaicloud/cruise-control:2.5.101"
+  - "ghcr.io/banzaicloud/cruise-control:2.5.123"
```

Comment:
How is this actually consumed by users deploying koperator - does this need to be updated in actual service ? Perhaps an override in https://github.com/mesosphere/dkp-catalog-applications/tree/main/services/kafka-operator/0.25.1/defaults (not sure, havent looked at upstream chart)

Reply:
maybe a docs update to point users to https://github.com/banzaicloud/koperator/blob/v0.25.1/charts/kafka-operator/templates/crds.yaml#L1396

Reply:
docs added https://d2iq.atlassian.net/wiki/spaces/DENT/pages/29919406/Kafka+in+a+Workspace